lib/prompt.mli

```diff
@@ -11,3 +11,7 @@ val confirm_or_abort :
   question:('a, unit) Logs.msgf -> yes:bool -> (unit, Rresult.R.msg) result
 (** Same as [confirm] but returns [Ok ()] for yes and
     [Error (`Msg "Aborting on user demand")] for no *)
+
+val confirm_or_abort_neg :
+  question:('a, unit) Logs.msgf -> yes:bool -> (unit, Rresult.R.msg) result
+(** Like [confirm_or_abort], but defaults to no when hitting enter *)
```

Comment:
What do you think of exporting the type `type choice = Yes | No` so that `?default_answer` can be a parameter of `confirm`/`confirm_or_abort` instead of adding a new function?

Reply:
Yes, I that's a good idea. I was trying to mess as little as possible with already written functions, but I think you're right.

As a site-note: This is quite a mess. Sorry! That commit you're referring to was part of my last PR  (and shouldn't be part of this one): https://github.com/ocamllabs/dune-release/pull/226/commits/149578a3c2da42bdfe7cf06acaa809c64eaac78f
I'll remove that commit from this PR and implement your suggestion in a separate one, if that makes sense.

Reply:
See #233